inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilderTest.java

```diff
@@ -138,6 +167,83 @@ public void setup()
         // https://dbpedia.org/sparql
         dbpedia = new SPARQLRepository("https://dbpedia.org/sparql");
         dbpedia.initialize();
+        
+        // http://nlp.dainst.org:8888/sparql
+        hucit = new SPARQLRepository("http://nlp.dainst.org:8888/sparql");
+        hucit.initialize();
+    }
+    
+    @Test
+    public void testLimitToClasses() throws Exception
```

Comment:
It is really difficult to understand what these tests are testing. I normally write the test method names more verbose so that it is easier to grasp and I add a describe. If these tests fail, then noone except hopefully you knows what this test should do. This is the same for many tests here.

Reply:
How would you call it? It tests whether `limitToClasses` works...

Reply:
testThatQeuryingForClassesOnlyReturnsClasses

Reply:
Renamed some of the methods.